lang/en/coursework.php

```diff
@@ -83,7 +82,6 @@
 $string['allocationactions_help'] = "These buttons will automatically allocate all of the students so that you don't have to do it manually. 'Auto-allocate all' will wipe all the existing allocations (including any ones you set manually) and randomly allocate graders based on the strategy you ahve chosen. The other buttons give you the option of leaving existing choices in place. Any students who have already been graded will of course be left as they are.";
 $string['allocationcheckboxtitle'] = 'Include {$a} in the moderations set';
 $string['allocationenabled'] = 'Assessor allocation enabled';
-$string['allocationenabled_help'] = '';
```

Comment:
this line isn't empty -  https://github.com/ucl-isd/moodle-mod_coursework/blob/main/lang/en/coursework.php#L872

not sure what is going on here. maybe comma needs removing

Reply:
Yes this is the help text for the Marking workflow / Assessor allocation enabled field and the comma should not be there.  I will remove it